Can't build the xtext plugin.

The log is like follow. I'll send PR for fixing this soon.

```
OpenJDK 64-Bit Server VM warning: If the number of processors is expected to increase from one, then you should configure the number of parallel GC threads appropriately using -XX:ParallelGCThreads=N
> Task :io.typefox.examples.theia.states:generateXtextLanguage UP-TO-DATE
> Task :io.typefox.examples.theia.states:generateXtext UP-TO-DATE
> Task :io.typefox.examples.theia.states:compileJava UP-TO-DATE
> Task :io.typefox.examples.theia.states:processResources UP-TO-DATE
> Task :io.typefox.examples.theia.states:classes UP-TO-DATE
> Task :io.typefox.examples.theia.states:jar UP-TO-DATE
> Task :io.typefox.examples.theia.states:assemble UP-TO-DATE
> Task :io.typefox.examples.theia.states:generateTestXtext UP-TO-DATE
> Task :io.typefox.examples.theia.states:compileTestJava UP-TO-DATE
> Task :io.typefox.examples.theia.states:processTestResources NO-SOURCE
> Task :io.typefox.examples.theia.states:testClasses UP-TO-DATE
> Task :io.typefox.examples.theia.states:test UP-TO-DATE
> Task :io.typefox.examples.theia.states:check UP-TO-DATE
> Task :io.typefox.examples.theia.states:build UP-TO-DATE

> Task :io.typefox.examples.theia.states.ide:generateXtext FAILED
ERROR:The method needsServerLayout(boolean) is undefined for the type LanguageAwareDiagramServer (file:/projects/theia-xtext-sprotty-example/language-server/io.typefox.examples.theia.states.ide/src/main/java/io/typefox/examples/theia/states/ide/diagram/StatesDiagramServerFactory.xtend line : 15 column : 11)

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':io.typefox.examples.theia.states.ide:generateXtext'.
> Xtext validation failed, see build log for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
9 actionable tasks: 1 executed, 8 up-to-date
```

--- a/language-server/io.typefox.examples.theia.states.ide/src/main/java/io/typefox/examples/theia/states/ide/diagram/StatesDiagramServerFactory.xtend
+++ b/language-server/io.typefox.examples.theia.states.ide/src/main/java/io/typefox/examples/theia/states/ide/diagram/StatesDiagramServerFactory.xtend
@@ -1,5 +1,6 @@
 package io.typefox.examples.theia.states.ide.diagram
 
+import org.eclipse.sprotty.ServerLayoutKind
 import org.eclipse.sprotty.xtext.DiagramServerFactory
 import org.eclipse.sprotty.xtext.LanguageAwareDiagramServer
 
@@ -12,7 +13,7 @@ class StatesDiagramServerFactory extends DiagramServerFactory {
 	override createDiagramServer(String diagramType, String clientId) {
 		val server = super.createDiagramServer(diagramType, clientId)
 		if (server instanceof LanguageAwareDiagramServer)
-			server.needsServerLayout = true
+			server.serverLayoutKind = ServerLayoutKind.AUTOMATIC
 		server
 	}
 }

